setup.py

```diff
@@ -60,6 +55,12 @@ def read(*filenames, **kwargs):
             "numpydoc>=0.6",
             "sphinx>=1.8",
         ],
+        "tests": [
+            "coverage>=4.3",
```

Comment:
Yeah, I find it useful to include all the packages we use when running the test suite though, not just the ones directly needed to run the tests.  Otherwise you end up defining some of your dependencies here and some in the script files in Travis-CI and it's just really confusing keeping track of that and keeping it in sync.  Since no one other than developers are likely to be installing the test requirements, it seems alright to me to add some extra dependencies there if it makes development easier.